src/index.html

```diff
@@ -1,12 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <ul class="calendar__container calendar__container--month-length--28 calendar__container--start-day--fri">
```

Comment:
```suggestion
    <ul class="calendar calendar--month-length--28 calendar--start-day--fri">
```
Modifiers should be added to block, not some element. Container is redundant here
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/a5bffee3-d947-4fec-8292-113081a4db42)